src/Conf.ml

```diff
@@ -830,6 +831,13 @@ module Formatting = struct
       (fun conf x -> {conf with parens_tuple= x})
       (fun conf -> conf.parens_tuple)
 
+  let parse_docstrings =
+    let doc = "Parse and format docstrings." in
+    let names = ["parse-docstrings"] in
+    C.flag ~default:true ~names ~doc ~section
```

Comment:
@gpetiot I think that this should default to `false` initially. We are seeing that enabling it can require changing the source docstrings, and furthermore when this happens it is reported as an internal error. This is a reasonably significant potentially-breaking change, so to ease the upgrade path, this should default to `false` for now, despite what we think ought to be the eventual default set of options.